docs/notebooks/hierarchical_binomial_bambi.ipynb

```diff
@@ -354,11 +362,11 @@
     "# Create handles for the legend (just dots and labels)\n",
     "handles = [\n",
     "    Line2D(\n",
-    "        [0], [0], label=\"At Bat\", marker=\"o\", color=\"None\", markeredgewidth=0,\n",
+    "        [0], [0], label=\"Hits\", marker=\"o\", color=\"None\", markeredgewidth=0,\n",
     "        markerfacecolor=RED, markersize=12\n",
     "    ),\n",
     "    Line2D(\n",
-    "        [0], [0], label=\"Hits\", marker=\"o\", color=\"None\", markeredgewidth=0, \n",
+    "        [0], [0], label=\"At Bat\", marker=\"o\", color=\"None\", markeredgewidth=0, \n",
```

Comment:
If we change the label name, then we also need to change the ordering of the scatter plots above. I think this bug can be solved by changing the `color` kwarg in `Line2D` to appropriately match the `color` kwarg of the scatter plot.

Reply:
The latest commit standardizes the order to be "Hits" and then "At Bat" so that the legend labels appear in the same order as the points